### PR TITLE
Run itests in a docker container

### PR DESCRIPTION
**Pull request by keshavdv**
Run chronos related itests in a docker container with all the reqs available

Thanks to @solarkennedy for most of the code that was copied over from the marathon-python project

### CODE REVIEW

File: itests/start-chronos.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 sudo /etc/init.d/zookeeper start
 sleep 5
 sudo /etc/init.d/chronos start
-sleep 5
+sleep infinity
```
Comment (solarkennedy) [inline itests/start-chronos.sh]: This can't sleep forever, it needs to return right away to allow the next step (tox) to go

Comment (keshavdv) [inline itests/start-chronos.sh]: Fixed

**Comment by solarkennedy** [comment]
Looks good! And now a green check-mark.

This will be useful to test against api changes with new chronos versions, and it will allow us to run itests with just docker and not installing chronos on my local laptop!

**Comment by asher** [comment]
Nice!

**Pull request merged**